modules/virt-disable-individual-bootsource-update.adoc

```diff
@@ -5,13 +5,17 @@
 
 :_content-type: PROCEDURE
 [id="virt-disable-individual-bootsource-update_{context}"]
-= Disabling an automatic update for a custom boot source
+= Disabling an automatic update for a single system-defined or user-defined boot source
 
-You can reduce the number of logs on disconnected environments or reduce resource usage by disabling the automatic imports and updates for an individual custom boot source.
+System-defined boot sources are not listed by default in the `spec.dataImportCronTemplates` of the `HyperConverged` custom resource (CR).
+In order to disable automatic imports and updates for a system-defined boot source, you add the boot source to the `dataImportCronTemplates:` field in the custom resource list and edit the `dataimportcrontemplate.kubevirt.io/enable` annotation.
```

Comment:
This line seems redundant. We already have this information in Procedure section.